dbt_af/operators/sensors.py

```diff
@@ -405,24 +404,18 @@ def __init__(
         self.target_environment = target_environment or dbt_af_config.dbt_default_targets.default_for_tests_target
         self.dbt_af_config = dbt_af_config
 
-        if kwargs.get('retries') and wait_timeout:
-            raise ValueError('You can not specify both wait_timeout and retries')
-        if not kwargs.get('retries') and not wait_timeout:
-            wait_timeout = _DEFAULT_WAIT_TIMEOUT
-
-        _poke_interval = kwargs.get('poke_interval', _DEFAULT_POKE_INTERVAL_SECONDS)
-        _retries = kwargs.get('retries', wait_timeout // _poke_interval - 1)
-        _airflow_task_timeout = max(_DEFAULT_WAIT_TIMEOUT, wait_timeout)
+        retry_policy = dbt_af_config.retries_config.dbt_source_freshness_retry_policy.as_dict()
+        retry_policy['retries'] = _DEFAULT_WAIT_TIMEOUT // _DEFAULT_POKE_INTERVAL_SECONDS - 1
+        retry_policy['poke_interval'] = _DEFAULT_POKE_INTERVAL_SECONDS
```

Comment:
Why do we rewrite `retires` and `poke_interval`? Maybe add logging when the old and the new values are not equal?

Reply:
Added logging that `retries` for source freshness operator are dynamically calculated